content/en/docs/ops/diagnostic-tools/component-logging/index.md

```diff
@@ -40,7 +40,7 @@ is `info` which is intended to provide the right amount of logging information f
 To control the output level, you use the `--log_output_level` command-line option. For example:
 
 {{< text bash >}}
-$ istioctl analyze --log_output_level attributes:debug,cli:warn
+$ istioctl analyze --log_output_level cli:warn
```

Comment:
Let's add a second scope back in to show the comma operation.

Reply:
Added. Thanks.